addons/web_editor/static/src/js/wysiwyg/widgets/chatgpt_alternatives_dialog.js

```diff
@@ -97,9 +100,11 @@ export class ChatGPTAlternativesDialog extends ChatGPTDialog {
                 }
             });
             messageIndex += 1;
+            this.state.isGenerating -= 1;
```

Comment:
The name `isGenerating` sounds like a boolean. Doing maths with booleans is evil. If it is a boolean, then don't do maths with booleans. If it's not, then rename the variable to something that does not sound like a boolean.

EDIT: Had a deeper look, it's indeed not a boolean, so please rename the variable. You know better than to use a name like that. I suppose it was done in a hurry or something, or maybe I'm reviewing the PR too quickly ?

Reply:
I just hesitated a lot. Originally it was a boolean, then finally I turned it into a number and I couldn't think of a better name without making it super long (and I figured "is generating 3" sounded right). But yeah it's icky. Maybe `leftToGenerate`, `numberLeftToGenerate`, `numberGenerating`, `messagesInProgress` ?